pypdf/_doc_common.py

```diff
@@ -0,0 +1,1303 @@
+# Copyright (c) 2006, Mathieu Fenniak
```

Comment:
Do we really need this header? As far as I can see, we have no consistent license headers, thus it should not really be required unless you want to explicitly declare your copyright here.

Reply:
_doc_common.py is mainly a copy/paste/extract of _reader.py and _writer.py. Not sure we should remove the copyright

Reply:
Thanks for the explanation. It does not really hurt, thus just keep it here.